scripts/marketplace_ci/sync.py

```
@@ -278,3 +280,38 @@ def apply_hooks_merge_plan(plan: HooksMergePlan) -> SyncResult:
+    staged_destinations: list[Path] = []
+    for action in actions:
+        if action.operation not in ("create", "update") or action.source is None:
+            continue
+        rel_source = action.source.resolve().relative_to(repo_resolved).as_posix()
+        if rel_source not in staged:
+            continue
+        # -f: a generated destination under .claude/.codex/.agents is a deliberately tracked
+        # file regardless of what a machine-local global gitignore says about those directory
+        # names (some machines exclude them everywhere) -- same override the test suite's own
+        # GitRepoHelper.stage() already applies for the identical reason.
+        subprocess.run(["git", "add", "-f", "--", str(action.destination)], cwd=repo, check=True)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)